src/rov/pca9685/src/main.cpp

```diff
@@ -0,0 +1,9 @@
+#include <rclcpp/rclcpp.hpp>
+#include "pca9685/PCA9685_Node.hpp"
+
+int main(int argc, char ** argv) {
+    rclcpp::init(argc, argv);
+    rclcpp::spin(std::make_shared<PCA9685_Node>());
+    rclcpp::shutdown();
+    return 0;
+}
```

Comment:
I'll admit, I probably wouldn't have felt like restructuring this whole thing just to make it more nicely-structured, but I give you a 👍 for going through the effort.
Or at least, that's all that I can see that has changed. Have there many any changes to the functionality? I couldn't find any.

Reply:
No functionality changes, just an interactive executable.